.github/workflows/performance_test.yaml

```diff
@@ -0,0 +1,48 @@
+name: Performance Test
+
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  test:
+    runs-on: macOS-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: "3.22.0"
+          cache: true
+          # Manually Update this `key`
+          cache-key: "20230512"
```

Comment:
We should update the cache key (keep on forgetting) if we want to benefit from the caching mechanism

Reply:
Why is that?

Reply:
Otherwise it won't refresh the cache
https://github.com/subosito/flutter-action

Reply:
Looks like using version and channel in the cache key might be better. I'll make the change. Let me know what you think.

Reply:
Yes also possible this way, it is just a label